### PR TITLE
Lets you press Ctrl+H (or just H in hotkey mode) to stop pulling

### PR DESCRIPTION
**Pull request by deathride58**
:cl: deathride58
tweak: You can now press Ctrl+H to stop pulling, or simply H to stop pulling if you're in hotkey mode.
/:cl:

It's been how many years and you still have to reach to the other side of the keyboard or click the stop pulling button to quickly stop pulling things?
This is a straight-forward QoL change.

### CODE REVIEW

**Comment by lzimann** [comment]
this conflicts with paste, gotta find some other key

**Comment by mathbrook** [comment]
why has no one else thought to do this

**Comment by praisenarsie** [comment]
real human beans use Delete to stop pulling

**Comment by Cyberboss** [comment]
We already have the delete key

**Review by Cyberboss — DISMISSED** [review]
Undo all the whitespace changes in skin.dmf

**Comment by deathride58** [comment]
The delete key is all the way across the keyboard. You need two hands to press the delete key while moving, on top of some keyboards not even having the delete key in the first place.
Meanwhile, you can start pulling objects with one hand, resist with one hand, switch hands with one hand, etc etc. The stop pulling button is one of two frequently used HUD buttons lacking a hotkey that can be reached without moving your entire hand.

**Comment by Doppeltongue** [comment]
He has a point. This would be especially nice when playing as mobs who don't have a stop pulling button on the UI.

**Comment by MrStonedOne** [comment]
i should point out that attempting to pull what you are already pulling or attempting to pull anything out or range stops your pull.

**Comment by Doppeltongue** [comment]
Doesn't that attempt grabbing if you're pulling something that can be grabbed?

**Comment by deathride58** [comment]
@MrStonedOne Attempting to pull something you're already pulling does *not* stop the pull at all. Pulling things out of range isn't nearly as intuitive as having a simple "stop pull" button.

**Comment by Kerbin-Fiber** [comment]
Yeah deathride is right, pulling something you are already pulling doesn't stop pulling it. Using H would be really helpful.

**Comment by lzimann** [comment]
Pulling something you already pull always stopped the pull, if it doesn't work then it is most likely a bug.

File: interface/interface.dm [hunk 1]
```diff
@@ -116,6 +116,7 @@ Hotkey-Mode: (hotkey-mode must be on)
 \tt = say
 \to = OOC
 \tb = resist
+\t<B></B>h = stop pulling
```
Comment (duncathan) [inline interface/interface.dm]: what's with the empty tag?

**Comment by Qbopper** [comment]
trying to pull something you're already pulling does not work at all and it hasn't for ages, if that's intended behaviour it's been broken for so long that no one knew about it

File: interface/interface.dm (hunk 1 above)
Comment (deathride58) [reply interface/interface.dm]: It has to be done so byond doesn't try to treat the \th as an escape character. It's the same method that was used for the talk-wheel hotkey.

**Comment by MrStonedOne** [comment]
Me adding pulling again to stop pulling happened shortly before the grab massacre of 2016 and was likely a casualty of that.

**Comment by Qbopper** [comment]
rip, that's something that would be good to have

**Pull request merged**